sklearn/metrics/_plot/tests/test_plot_roc_curve.py

```diff
@@ -36,7 +36,7 @@ def test_plot_roc_curve_error_non_binary(pyplot, data):
     clf = DecisionTreeClassifier()
     clf.fit(X, y)
 
-    msg = "DecisionTreeClassifier should be a binary classifer"
+    msg = "DecisionTreeClassifier should be a binary classifier"
```

Comment:
I never been the best at spelling. ;)

Reply:
My editor as a spelling corrector otherwise I would not have see it :)